frameworks/javascript/cw-2.js

```diff
@@ -356,6 +356,12 @@ try {
         Test.expect(false, msg, options);
       }
     },
+    assertFuzzyEquals: function (actual, expected, msg = "") {
```

Comment:
The test is failing because of `SyntaxError`:

```
/runner/frameworks/javascript/cw-2.js:359
   assertFuzzyEquals: function (actual, expected, msg = "") {
                                                       ^
SyntaxError: Unexpected token =
```

I didn't realize this until I saw this, but the test framework needs to be compatible with Node 0.10.x.